src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -531,6 +532,16 @@ public DatasetSummary retrieveSummaryByName(String name) {
         if (!whereClauses.isEmpty()) {
             whereSql = " WHERE " + StringUtils.join(whereClauses, " AND ");
         }
+
+        // get the filtered total count of objects without offset and limit
```

Comment:
Why don't we want to include the limit in the filtered total?

Reply:
We want the filtered total, not the window of the filter that we're currently looking at. We need the total number that match the filter so that the UI knows to paginate and display the arrows.

Reply:
Oh I was thinking it would be confusing at the API level to see a list of datasets/snapshots where the total is less than the "filtered total" but I see what you mean now!